packages/liveblocks-client/rules/console-must-be-fancy.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  meta: {
+    messages: {
+      addMissingImport:
+        "Don't use `console` directly. To fix: `import * as console from './fancy-console'` in this module.",
+    },
+  },
+
+  create(context) {
+    let seenFancyImport = false;
+
+    return {
+      ImportDeclaration(node) {
+        if (node.source.value.includes("fancy-console")) {
+          seenFancyImport = true;
+        }
+      },
+
+      CallExpression(node) {
+        if (node.callee.object?.name === "console" && !seenFancyImport) {
+          context.report({ node: node.callee, messageId: "addMissingImport" });
+        }
+      },
+    };
+  },
+};
```

Comment:
Here is a custom ESLint rule I wrote to ensure that we don't accidentally use `console.log()`. Every module that uses a `console.xxx` statement will be reminded about doing that fancy import automatically.

Reply:
The reason this needed to be a full custom plugin is that it's a little more complicated rule than just disallowing a certain syntax. We cannot just disallow all console statements. "It depends" on whether the fancy console module has been imported or not. If it were simpler, I wouldn't have used a custom plugin, but instead disallowed the syntax in the .eslintrc file directly, using the `"no-restricted-syntax"` config.